packages/fxa-settings/src/components/App/index.test.tsx

```diff
@@ -3,23 +3,30 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React, { ReactNode } from 'react';
-import { act } from '@testing-library/react';
+import { act, screen, waitFor } from '@testing-library/react';
 import { renderWithLocalizationProvider } from 'fxa-react/lib/test-utils/localizationProvider';
 import App from '.';
 import * as Metrics from '../../lib/metrics';
 import {
   AppContext,
-  useAccount,
-  useInitialSettingsState,
+  useInitialMetricsQueryState,
+  useLocalSignedInQueryState,
   useIntegration,
 } from '../../models';
-import { createAppContext, mockAppContext } from '../../models/mocks';
+import {
+  createAppContext,
+  mockAppContext,
+  renderWithRouter,
+} from '../../models/mocks';
 import GleanMetrics from '../../lib/glean';
 import config from '../../lib/config';
+import * as utils from 'fxa-react/lib/utils';
+import { HomePath } from '../../constants';
```

Comment:
unused

Reply:
Ugh, thanks for catching these. Do you have something that auto-removes these for you or should we re-open [FXA-4125](https://mozilla-hub.atlassian.net/browse/FXA-4125) which I think would be nice to have?

Reply:
Nope, but VSCODE flags them and I usually notice in review. Opening FXA-4125 would be nice, cause this happens a lot to me also. VSCODE will sometimes automatically add incorrect import behind my back... and it's easy to miss.

Reply:
Re-opened!